edge/services/helloworld/MaintainHW.md

```diff
@@ -0,0 +1,18 @@
+# Process for the Horizon Development Team to Make Updates to the Helloworld Service
+
+Instructions for the open-horizon developers that are maintaining this example code:
+
+- Do the steps in the [README.md Preconditions](#preconditions) section, **except**:
+  - export `HZN_EXCHANGE_URL` to the staging instance
+  - Do **not** run `hzn dev service new ...` (use the git files in this directory instead)
+  - export `HZN_EXCHANGE_USER_AUTH` to your credentials in the IBM org
+- Make whatever code changes are necessary
```

Comment:
Will the reader know what (if any) code changes are necessary? How would they know?

Reply:
Since this is only for our dev team, yes they will know